nrrd-design-system/components/components/links/links.hbs

```diff
@@ -12,9 +12,12 @@
 
   <!-- new: link-icon-doc -->
   <div class="container-margin">
-    <a href="/downloads/federal-production/">
-      <icon class="fa fa-file-text-o u-padding-right"></icon>Text link with document icon
-    </a>
+    <p>
+      <a href="{{site.baseurl}}/downloads/#all-lands-and-waters">
+        <object type="image/svg+xml" data="{{site.baseurl}}/public/img/icons/file-text-o.svg" class="u-padding-right icon-small">
```

Comment:
I'm not seeing this icon display here: 

![screen shot 2018-04-06 at 3 18 50 pm](https://user-images.githubusercontent.com/11636908/38440052-0631af88-39ae-11e8-8adf-900ce0388d99.png)

Reply:
This one is a mystery. I'll work on resolving it next week.